### PR TITLE
Automod Notifications

### PR DESCRIPTION
**Pull request by jvc56**

### CODE REVIEW

File: pkg/mod/mod.go [hunk 1]
```diff
@@ -322,66 +315,7 @@ func applyAction(ctx context.Context, us user.Store, cs user.ChatStore,
...
-					} else if resp.StatusCode != 204 { // No Content
-						// We do not expect any other response
-						log.Err(err).Str("status", resp.Status).Msg("mod-action-discord-notification-post-bad-response")
-					}
-				}
-			}
-		}
-	}
+	notify(ctx, us, user, action)
 	return us.Set(ctx, user)
```
Comment (andy-k) [inline pkg/mod/mod.go:319]: not sure if `notify` should be before `us.Set` succeeds

File: pkg/mod/automod.go [hunk 2]
```diff
@@ -175,11 +175,13 @@ func updateNotoriety(ctx context.Context, us user.Store, ns NotorietyStore, user
...
-			err = setCurrentAction(user, &ms.ModAction{UserId: user.UUID,
+			action := &ms.ModAction{UserId: user.UUID,
 				Type:          ms.ModActionType_SUSPEND_GAMES,
 				StartTime:     ptypes.TimestampNow(),
 				ApplierUserId: AutomodUserId,
-				Duration:      int32(DurationMultiplier * (newNotoriety - NotorietyThreshold))})
+				Duration:      int32(DurationMultiplier * (newNotoriety - NotorietyThreshold))}
+			err = setCurrentAction(user, action)
+			notify(ctx, us, user, action)
 			if err != nil {
```
Comment (andy-k) [inline pkg/mod/automod.go:185]: not sure if `notify` should be before `setCurrentAction` and `us.Set` succeed

File: pkg/mod/mod.go (hunk 1 above)
Comment (jvc56) [reply pkg/mod/mod.go:319]: `notify` after `us.Set` makes more sense. I'll update it.

File: pkg/mod/automod.go (hunk 2 above)
Comment (jvc56) [reply pkg/mod/automod.go:185]: `notify` after `us.Set` makes more sense. I'll update it.

**Review by andy-k — APPROVED** [review]

**Pull request merged**